wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -191,6 +188,62 @@
                 }
               }
             },
+            {
+              "if": {
+                "properties": {
+                  "type": {
+                    "enum": [
+                      "cheEditor"
+                    ]
+                  }
+                }
+              },
+              "then": {
+                "required": [
+                  "id"
+                ],
+                "additionalProperties": false,
+                "properties": {
```

Comment:
Wouldn't it be useful to add the prefs also to the editor conceivably?

Reply:
I know nothing about preferences for Theia plugins but I asked @akurinnoy and he did not see sense in it.
@evidolob @l0rd @benoitf Do you think preferences can be useful for cheEditor component as well?

Reply:
It may be added later if needed